src/main/java/GeneralService/RequestHandlers/WindowControlRequestHandler.java

```diff
@@ -0,0 +1,29 @@
+package GeneralService.RequestHandlers;
+
+import GeneralService.Request;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class WindowControlRequestHandler implements RequestHandler {
+
+    private static final String MQTT_WINDOW_SERVICE_TOPIC = "WindowControlService";
+    @Autowired
+    private MqttClient mqttClient;
+
+    public WindowControlRequestHandler() {
+    }
+
+    @Override
+    public void sendRequest(Request request) throws MqttException {
+        if (!mqttClient.isConnected())
+            mqttClient.connect();
+
```

Comment:
before actually publishing, make sure that the requestType is valid, check it against the requestTypes you added to that other class